Improve Readme.md (#46)
thgh/vercel-sapper: README.md

Check out this [demo project](https://github.com/beyonk-adventures/now-sapper-demo) that uses this builder. It can be used as a template, or a way to verify correct usage of the following instructions.

##### 1. Install `vercel-sapper` as part of `devDependencies`

```bash

>  npm install -D vercel-sapper

Reviewer: You don't need to install the builder as a dev dependency. If you do so, Vercel will simply ignore it and install its own copy from npm.

Author: Thanks. Did not know that. I have updated it.

  "builds": [{ "src": "package.json", "use": "vercel-sapper" }]
}
```

> Note: You don't have to install it as any dependency in your project. Vercel build process will install it from npm. More about [builds](https://vercel.com/docs/configuration#project/builds).

Reviewer: ```suggestion
Note: Don't install `vercel-sapper` it as a dependency in your project. The Vercel build process will automatically install it from npm. More about [builds](https://vercel.com/docs/configuration#project/builds).
```
